plugins/outputs/stackdriver/stackdriver.go

```diff
@@ -8,28 +8,17 @@ import (
 	"sort"
 	"strings"
 
-	monitoring "cloud.google.com/go/monitoring/apiv3/v2" // Imports the Stackdriver Monitoring client package.
-	"google.golang.org/api/option"
-	metricpb "google.golang.org/genproto/googleapis/api/metric"
-	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
-	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
-	"google.golang.org/protobuf/types/known/timestamppb"
-
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/outputs"
-)
-
-// Stackdriver is the Google Stackdriver config info.
-type Stackdriver struct {
-	Project        string
-	Namespace      string
-	ResourceType   string            `toml:"resource_type"`
-	ResourceLabels map[string]string `toml:"resource_labels"`
-	Log            telegraf.Logger   `toml:"-"`
+	"google.golang.org/api/option"
+	"google.golang.org/genproto/googleapis/api/metric"
+	"google.golang.org/genproto/googleapis/api/monitoredres"
 
-	client *monitoring.MetricClient
-}
+	mon "cloud.google.com/go/monitoring/apiv3/v2"
+	monpb "google.golang.org/genproto/googleapis/monitoring/v3"
+	tspb "google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Can you please revert renaming the aliases to reduce the diff to a minimum. This eases reviews. Feel free to submit renaming in an additional PR.

Reply:
Fair enough.  I'm going to leave them as-is in counter_cache.go unless you object.